### ISSUE TITLE
SQuAD output layer and the computation loss

### ISSUE THREAD

**Issue by tsaikenyoo** [huggingface/pytorch-pretrained-BERT#214]
Hi, I noticed that in the final linear layer of `BertForQuestionAnswering`, the loss is computed based on `start_logits` and `end_logits `. That means the positions of questions are also considered to compute loss. Maybe we should only care about the positions of context? e.g., by setting the question part of `start_logits` and `end_logits ` to `-inf`?

https://github.com/huggingface/pytorch-pretrained-BERT/blob/0a9d7c7edb20a3e82cfbb4b72515575543784823/pytorch_pretrained_bert/modeling.py#L1089-L1113

**Comment by thomwolf** [comment]
Yes you can also do that.
